pkgs/apis/v1/types_originissuer.go

```diff
@@ -62,6 +97,8 @@ type OriginIssuerAuthentication struct {
 type SecretKeySelector struct {
 	// Name of the secret in the OriginIssuer's namespace to select from.
 	Name string `json:"name"`
+	// Namespace of the secret in the OriginIssuer's namespace to select from.
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
I don't think you should have this field, in existing `ClusterIssuer` resources (eg `apiKeySecretRef`) the Secret needs to be in the same namespace as the controller pod. This can be overridden in cert-manager itself with `--cluster-resource-namespace` to the controller.

Reply:
I'm happy to remove it if that's the recommendation?  I thought it'd be more flexible and can't see an attack vector, but not being able to specify the namespace would suit our use cases just fine.